### PR TITLE
Add ko.json

### PR DESCRIPTION
**Pull request by hs7982**
hs7982#6039

Added Korean translation.

+ May I ask you a question? Some people, including myself in our country, are getting errors on their homepages. The error message is **"Unable to retrieve items, please try again in a few moments."** Is there a solution?

### CODE REVIEW

File: ko.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+        },
+        "randomiser" : {
+            "title" : "랜덤 아이템",
+            "line1" : "사용 가능한 모든 의상 중 무작위로 보여줍니다.",
+            "chooseText" : "클릭하여 선택하세요",
+            "randomise-all" : "모두 섞기"
+        },
+        "png" : {
+            "title" : "치장 아이템 PNG",
+            "line1" : "Please credit back to fnbr.co if you use any of these images. Previews are scaled down, downloads will be full size.",
```
Comment (Krymonota) [inline ko.json]: Would it be possible to translate this string?

File: ko.json [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+            "sort-by" : "정렬 기준:",
+            "all" : "모두"
+        },
+        "gallery" : {
+            "title" : "치장 아이템 갤러리",
+            "line1" : "포트나이트 배틀로얄에서 사용할 수 있는 모든 치장 아이템입니다.",
+            "line2" : "Please do not use these images on your personal website/social media account without direct credit to fnbr.co"
```
Comment (Krymonota) [inline ko.json]: Would it be possible to translate this string?

**Comment by Krymonota** [comment]
This is an interesting error, but unfortunately I don't have a way to check or fix it. It's probably best to report it to the #support channel on our Discord server.

File: ko.json [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+                "line3" : "또는, [디스코드로 로그인](/oauth/discord) 할 수 있습니다."
+            }
+        },
+        "errors" : {
+            "404" : {
+                "title" : "에러",
+                "line1" : "요청한 리소스가 없거나, 볼 수 있는 권한이 없습니다."
+            },
+            "500" : {
+                "title" : "Uh oh!",
```
Comment (Krymonota) [inline ko.json]: Just missed this one.

**Comment by Krymonota** [comment]
This is an interesting error, but unfortunately I don't have a way to check or fix it. It's probably best to report it to the #support channel on our Discord server.

File: ko.json (hunk 3 above)
Comment (hs7982) [reply ko.json]: This line is not translated as it is used a lot in Korea.

File: ko.json (hunk 2 above)
Comment (hs7982) [reply ko.json]: End.

File: ko.json (hunk 1 above)
Comment (hs7982) [reply ko.json]: End.

**Comment by hs7982** [comment]
All requested translations have been completed.

**Pull request merged**